app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
```

Comment:
I don't recommend doing this in most cases. It's making assumptions about future development instead of sticking to the single feature you're working on. It could happen that you pivot and never make those routes (probably not for this project, but if you were working on a live product that could totally happen!), so there's confusing unused code hanging out forever

If you keep it in here, it's your responsibility to make sure all those routes get used and track if they are not. You are the owner of that code. You will need to explain why it's there in a week if it goes unused. Git blame remembers. 😄